packages/react-theme/src/utils/index.ts

```diff
@@ -0,0 +1,2 @@
+export { mergeThemes } from './mergeThemes';
```

Comment:
Hmm, utils are not private anymore :-/
In that case I would remove this index and export the two directly in the main index.